frontend/src/metabase/query_builder/components/GuiQueryEditor.jsx

```diff
@@ -212,7 +212,11 @@ export default class GuiQueryEditor extends Component {
                     );
                 }
             }
-            return aggregationList
+            return (
+              <div className={cx("Query-section Query-section-breakout scroll-x scroll-show scroll-show--horizontal")}>
```

Comment:
I'm not sure we need the `Query-section-breakout` class here since this is the aggregation list.

Reply:
@kdoh I had put that in there to give the aggregations a parent div and was was playing around with trying to get the scroll on this parent (which affected both the query section and the aggregation). I understand that it's misnamed and I can adjust that. I can remove the name and merge the changes from master into this before continuing to work on it. Thanks!